service/src/main/java/com/ericsson/eiffel/remrem/generate/constants/RemremGenerateServiceConstants.java

```diff
@@ -35,4 +35,9 @@
         public static final String UNAVAILABLE_FOR_FAILIFNONEFOUND = "{\"status_code\": 406, \"result\": \"FAIL\", "
                 + "\"message\":\"No event id found with ERLookup properties\"}";
 
+        public static final String PAGESIZE = "The number of events to be displayed per page.";
```

Comment:
i agreed it won't support pagination so I will change the "The number of events to be displayed per page"  to "Number of events to be displayed"
please check the below link  about query parameters available in ER
https://wcdma-confluence.rnd.ki.sw.ericsson.se/pages/viewpage.action?spaceKey=EIFFEL&title=ER+Query+options+and+its+usage 

pageSize means number of events, actually this value by default one. 
yes it will truncate the ER lookups results if its have the multiple results so user has to configure the value as he wants

Reply:
@xsudban, we should not reference Ericsson internal websites on GitHub, as most readers will not be able to access them. Anyway, I believe that reference doesn't add much to this discussion anyway. It's good that you update the text for the pageSize parameter.